src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CachingFilePathComparer.cs

```diff
@@ -51,61 +50,19 @@ public bool Equals(string? x, string? y)
 
         public int GetHashCode([DisallowNull] string obj)
         {
-            if (TryGetCachedHashCode(obj, out var hashCode))
-                return hashCode;
+            // Stephen thinks this may be faster on NetFx as it will help the runtime with reading/writing from a single location.
```

Comment:
We have a lot of Stephens around here, you might want to do a full name. :smile: